test/models/tasks/copy_media_task_test.rb

```diff
@@ -45,8 +45,12 @@
     assert_equal url, "s3://test-prx-feed/path/guid/audio.mp3"
   end
 
-  it "use original url as the source url" do
+  it "use original url as the source url until complete" do
+    task.media_resource.status = "started"
```

Comment:
I want to say that this would use the `original_url` until the actual process of re-evaluating the `task.media_resource` is finished so it can decide if it wants to use that `url` first as the source before falling back to the `original_url`.